tests/state_test_util.go

```diff
@@ -122,6 +123,28 @@ type stTransactionMarshaling struct {
 	PrivateKey           hexutil.Bytes
 }
 
+//go:generate gencodec -type stAuthorization -field-override stAuthorizationMarshaling -out gen_stauthorization.go
+
+// Authorization is an authorization from an account to deploy code at it's
+// address.
+type stAuthorization struct {
+	ChainID *big.Int
+	Address common.Address `gencodec:"required" json:"address"`
+	Nonce   uint64         `gencodec:"required" json:"nonce"`
+	V       *big.Int       `gencodec:"required" json:"v"`
+	R       *big.Int       `gencodec:"required" json:"r"`
+	S       *big.Int       `gencodec:"required" json:"s"`
+}
+
+// field type overrides for gencodec
+type stAuthorizationMarshaling struct {
+	ChainID *math.HexOrDecimal256
```

Comment:
https://github.com/ethereum/go-ethereum/pull/30078/files#diff-0f2a03d410f3b42fb5cc232db0be82b42a741b952aa3999887370ef91c1eca58R156
math.HexOrDecimal64

Reply:
Kaia treats ChainId and V as a big.Int.
Eth has inconsistencies with fields of the same name, should we follow this?

This is an Eth specification.
> assert auth.chain_id < 2**64
assert auth.nonce < 2**64
assert len(auth.address) == 20
assert auth.y_parity < 2**8
assert auth.r < 2**256
assert auth.s < 2**256

Reply:
Yes, it doesn't matter how each value is treated internally. We need to validate if the value of the transaction follows spec.

Reply:
Make sense, it seems that we need to change the authorization as well. Thank you.

Reply:
It's fixed by https://github.com/kaiachain/kaia/pull/145/commits/7955b5213874033a5a0fed07e4c05f7668fbf167 .
Please check it!